clusterloader2/testing/network/config.yaml

```diff
@@ -0,0 +1,33 @@
+{{$PROTOCOL := .PROTOCOL}}
+{{$NUMBER_OF_SERVERS := .NUMBER_OF_SERVERS}}
+{{$NUMBER_OF_CLIENTS := .NUMBER_OF_CLIENTS}}
+
+name: network
+automanagedNamespaces: 1
+tuningSets:
+- name: Sequence
+  parallelismLimitedLoad:
+    parallelismLimit: 1
+steps:
+- name: start network metric measurement
+  measurements:
+    - Identifier: NetworkPerformanceMetrics
+      Method: NetworkPerformanceMetrics
+      Params:
+        action: start
+        duration: 10s
```

Comment:
Only 10s?

Reply:
yes we just started with this value. Once the entire code is ready and it starts running in prow job , we can decide the runtime of each of these testcases. It has an impact on total runtime.